ci/ci/build.py

```diff
@@ -833,22 +833,24 @@ def config(self, scope):  # pylint: disable=unused-argument
         }
 
     def build(self, batch, code, scope):  # pylint: disable=unused-argument
-        if scope == 'dev':
-            return
```

Comment:
So I think this should stay.  If we're deploying in dev, we shouldn't do anything.  The constructor code sets the step configuration to use the developer's already existing database.  We should never create another database for the developer.

Reply:
ok. This will make it harder to setup the test database, but I can put in more infrastructure for that.

Reply:
I'm confused.  The dev database must always be pre-setup.

Reply:
Let me say this another way: in the dev scope, the database server is "virtualized" by the developer's database.  Instead of creating new databases, we just reuse the developer's database.  The different databases aren't isolated, but that's fine because this is just for development.  Does that make sense?

Reply:
Sorry, I wasn't clear. If I want to add a new **instance**, then the database isn't setup for my dev account at all. That's because Alex's create users script does some setup, which I won't have in the case of a brand new SQL instance.

Reply:
Ah, I see.  Yes, we'll want a tool so you can set up your dev namespace for the new instance, and then you'll be golden.  Basically, you need to set up a database and two users and create the sql-jigold-jigold-{user, admin}-config secrets.  You'll want to back up your secrets for the prod instance or re-run it against prod when you're done.  I don't think it's been thought through how to make all of this work nicely, I think that's something @akotlar wanted to do after ASHG.

Reply:
I think you can't remove this dev check in the general case, but you could disable it for specific deploys and/or steps, put a secret for your instance/server in batch-pods, and then run this build step against that.  Maybe that's what you were aiming at?